Exercise_A/index.html

```diff
@@ -17,29 +18,221 @@
+            <div class="container">
+                <div class="row">
+                    <div class="title col-md-12">
+                        <h1 class="text-center">My Contact Information</h1>
+                        <div class="line"></div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="lead col-md-12 d-flex flex-column align-items-center">
+                        <p class="text-center">Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies
+                            wanting my insane website design skills.</p><br>
+                        <p class="text-center">Honestly this template does look really nice, though.</p>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="contact col-md-4 offset-md-2 d-flex flex-column align-items-center">
+                        <i class="iconPhone fas fa-phone-alt wow animate__animated animate__bounceIn"></i>
+                        <p class="text-center">+1-608-317-1888</p>
+                    </div>
+
+                    <div class="contact col-md-4 d-flex flex-column align-items-center ">
+                        <i class="iconMail far fa-envelope wow animate__animated animate__bounceIn "></i>
+                        <a href="mailto: ">holy@shit.com</a> </div>
+                </div>
+            </div>
+        </footer>
+        </div>
+
+
+
+
+        <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+
+        <!-- 
       All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
-  </body>
-</html>
+
+
+
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js " integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj " crossorigin="anonymous "></script>
+
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js " integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN " crossorigin="anonymous "></script>
+
+
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js " integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p " crossorigin="anonymous "></script>
+
+        <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll@15/dist/smooth-scroll.polyfills.min.js "></script>
+
+
+        <script src="./assets/js/wow.js "></script>
+        <script type="text/javascript ">
+            new WOW().init()
+        </script>
+        <script src="./assets/js/script.js "></script>
+</body>
+
+</html>
```

Comment:
Amazing Job with the website, it works perfectly and also you have made good use of animations.